lib/ui/mail_view_indv.dart

```diff
@@ -1,24 +1,71 @@
+import 'dart:convert';
+import 'package:enough_mail/codecs.dart';
 import 'package:flutter/material.dart';
-import 'package:flutter/widgets.dart';
+import 'package:global_configuration/global_configuration.dart';
 import 'package:intl/intl.dart';
 import 'package:summer2022/models/MailPiece.dart';
 import 'package:summer2022/ui/bottom_app_bar.dart';
+import '../models/Digest.dart';
+import '../services/mail_retrieveByMailPiece.dart';
 
-class MailPieceViewWidget extends StatelessWidget{
+class MailPieceViewWidget extends StatefulWidget{
+
+  final MailPiece mailPiece;
+
+  const MailPieceViewWidget({Key? key, required this.mailPiece}) : super(key: key);
+
+  @override
+  MailPieceViewWidgetState createState() => MailPieceViewWidgetState(mailPiece);
+}
+
+  GlobalConfiguration cfg = GlobalConfiguration();
+
+  class MailPieceViewWidgetState extends State<MailPieceViewWidget> {
+  GlobalConfiguration cfg = GlobalConfiguration();
 
   final FontWeight _commonFontWeight = FontWeight.w500;
   final double _commonFontSize = 30;
   final Color _buttonColor = Color.fromRGBO(51, 51, 102, 1.0);
+  final mailPiece;
+  late Digest digest;
+  late Image mailImage = Image.asset('assets/mail.test.02.png');
```

Comment:
No more test data

Reply:
we will want to provide a default load image while the email is getting fetched and waiting for it to load.  Image.asset mail.test.02.png was just the default.  When you run the program, replaces the image from the email if it exists.

Reply:
I see this isn't clear, I mean we can pick a different default image, or just have it be null.

Reply:
Yeah that sounds like the right idea. I think either nothing or some loading spinner would be better than another mail item